src/helpers.js

```diff
@@ -58,3 +58,19 @@ export const getFunctionName = (type) => {
     }
     return typeName;
 };
+
+export const removeFirstKeyIfNumber = (keys) => {
+    const firstKey = parseInt(keys[0], 10);
+    if (Number.isInteger(firstKey)) {
+        keys.splice(0, 1);
+    }
+    return keys;
+};
+
+export const getErrorIndexFromKeys = (keys) => {
+    const firstKey = parseInt(keys[0], 10);
+    if (Number.isInteger(firstKey)) {
```

Comment:
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Number/isInteger - does not work on IE.
https://babeljs.io/learn-es2015/#math--number--string--object-apis - pollyfill is recommended.

Reply:
new helper function added. Pollyfill is heavy. Bundle file increase 100kB with pollyfill.